tests/infra/jsonrpc.py

```diff
@@ -313,8 +328,8 @@ def do(self, method, params, expected_result=None, expected_error_code=None):
             assert expected_error_code.value == r.error["code"]
         return r
 
-    def rpc(self, method, params):
-        id = self.request(method, params)
+    def rpc(self, method, params, readonly_hint=None):
+        id = self.request(method, params, readonly_hint)
```

Comment:
Minor: When we're just forwarding args in Python, and as the interemediary we don't need to know what they are called or what they do, we can write:
```
def foo(*args, **kwargs):
  x = bar(*args, **kwargs)
```
I think this might be the right pattern here where we have so many functions repeating and forwarding arguments, but I'll defer to Pythonians.

Reply:
The only drawback with that is you lose precision in the doc/IDE hints, which for internal test code (as opposed to a public API) is acceptable IMO.

Reply:
Done.